### ISSUE TITLE
Pihole:2022.01.1 returns incorrect result from api.php?status

### ISSUE THREAD

**Issue by diamondsw** [pi-hole/docker-pi-hole#983]
This is a: Bug

## Related Issues
- [x] I have searched this repository/Pi-hole forums for existing issues and pull requests that look similar 
I originally attributed this to issues [here](https://github.com/pi-hole/docker-pi-hole/issues/962#issuecomment-1004500755).

## How to reproduce the issue 
1. Start Pihole normally (in my case via compose file below)
2. Once started, note the Pi-hole web dashboard indicates Status: Active, and confirm it is processing DNS requests.
3. Query its status with `curl -s GET 'http://pihole.joshuaochs.com/admin/api.php?status'`
4. Result should be `{"status":"enabled"}`, but API returns `{"status":"disabled"}`

## Environment 
1. Environment data
  * Operating System: Debian 10.11 
  * Hardware: Dell PowerEdge R720xd and Raspberry Pi 3b
  * Kernel Architecture: amd64 (Dell) and Arm (Pi)
  * Docker Install Info and version: 
    - Software source: Docker version 20.10.12, build e91ed57; Official docker-ce from Docker's debian repo
    - Supplementary Software: docker-compose version 1.29.2, build 5becea4c 
  * Hardware architecture: x86 (Dell) and Arm (Pi)

2. docker-compose.yml
```
version: "3"
services:
  pihole:
    image: pihole/pihole:latest
    container_name: pihole
    hostname: 'docker.homelab'
    restart: unless-stopped
    environment:
      TZ: 'America/Chicago'
    volumes:
      - ./dnsmasq:/etc/dnsmasq.d:rw
      - ./pihole:/etc/pihole:rw
    dns:
      - 1.1.1.1
    networks:
      macvlan_subnet:
        ipv4_address: 10.0.1.229

networks:
  macvlan_subnet:
    external: true
```

3. any additional info to help reproduce

## These common fixes didn't work for my issue

- [x] I have tried removing/destroying my container, and re-creating a new container
- [ ] I have tried fresh volume data by backing up and moving/removing the old volume data
- [ ] I have tried running the stock `docker run` example(s) in the readme (removing any customizations I added)
- [ ] I have tried a newer or older version of Docker Pi-hole (depending what version the issue started in for me)
- [ ] I have tried running without my volume data mounts to eliminate volumes as the cause

I confess that I have *not* tried clearing my config as this occurs on two separate machines/instances, and the API does not seem like it would be configuration dependent, but if there is any issue reproducing this at all then I certainly can.

**Comment by rdwebdesign** [comment]
Duplicate of pi-hole/AdminLTE#2051, pi-hole/AdminLTE#2053, pi-hole/AdminLTE#2056, pi-hole/AdminLTE#2064

Fixed by pi-hole/AdminLTE#2052

**Comment by diamondsw** [comment]
Apologies; I was looking at issues in the wrong project!
